README.md

```diff
@@ -67,6 +81,22 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 	- [Russian](#russian)
 
 
+Icons and tags:
+
+- Ancient: ![ancient]
```

Comment:
Not really archived, awesome java use this description:

In existence since the beginning of time, and which people will continue using long after the hype has waned.